.github/workflows/prebuild.yml

```diff
@@ -0,0 +1,32 @@
+name: prebuild
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Since this attempts to upload the artifacts to a release (`prebuild -u`), we only want to run this on tags, not on every push to master. It looks like this is where it is failing on the [run on your branch](https://github.com/thegecko/node-usb/runs/1697564231).

We do want something that builds the C++ on at least one version of Node on every branch and pull request. I'm not super familiar with GHA, but maybe that belongs in a separate .yml file vs using conditionals like we did on Travis?